src/components/icon/icon.css

```
@@ -33,15 +33,17 @@ svg {
   width: 100%;
 }
 
-
 /* Icon RTL
  * -----------------------------------------------------------
  */
 
-:host(.flip-rtl) .icon-inner {
+:host(.flip-rtl):host-context([dir='rtl']) .icon-inner {
   transform: scaleX(-1);
 }
 
+:host(.flip-rtl:dir(rtl)) .icon-inner {
+  transform: scaleX(-1);
+}
```

```suggestion
@supports selector(:dir(rtl)) {
  :host(.flip-rtl:dir(rtl)) .icon-inner {
    transform: scaleX(-1);
  }
}
```
Just to be safe, can we wrap this in an `@supports` check which is what the Framework mixin does?